test/integration/integration_suite_test.go

```diff
@@ -65,7 +64,9 @@ var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter)))
 
 	if v, ok := os.LookupEnv(dumpLogsEnvName); ok {
-		dumpLogs = strings.ToLower(v) == "true"
+		if shouldDumpLogs, err := strconv.ParseBool(v); err != nil {
+			dumpLogs = shouldDumpLogs
+		}
```

Comment:
Probably you wan to set it only when `err == nil`?
Anyway, the `strconv.ParseBool` returns false in case of error, this is the reason why I proposed to ignore the error.

Reply:
My mistake. Will fix it now.